addons/pharma/functions/fnc_treatmentAdvanced_TXALocal.sqf

```diff
@@ -0,0 +1,65 @@
+#include "script_component.hpp"
+/*
+ * Author: 2LT.Mazinski
+ * Begins TXA bandaging process
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Medication <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, "TXA"] call kat_circulation_fnc_treatmentAdvanced_TXALocal;
+ *
+ * Public: No
+ */
+
+params ["_target", "_item"];
+
+[_target, _item] call ace_medical_treatment_fnc_addToTriageCard;
+[_target, "activity", LSTRING(push_log), [[_medic] call ace_common_fnc_getName, "TXA"]] call ace_medical_treatment_fnc_addToLog;
+[_target, "TXA", 5, 120, 0, 0, 0] call ace_medical_status_fnc_addMedicationAdjustment;
+
+
```

Comment:
Also add this behavior for the pfh `if (_unit getVariable ["kat_TXA_PFH", false]) exitWith {};
_unit setVariable ["kat_TXA_PFH", true];`

Reply:
Check that I did this right please.